rotkehlchen/db/dbhandler.py

```diff
@@ -3416,3 +3417,37 @@ def delete_web3_node(self, identifier: int) -> None:
                 proportion_to_share=ONE,
                 exclude_identifier=None,
             )
+
+    def get_all_user_notes(self) -> List[UserNote]:
+        """Returns all the notes created by a user."""
+        with self.conn.read_ctx() as cursor:
+            cursor.execute('SELECT identifier, title, content, location, creation_timestamp FROM user_notes')  # noqa: E501
+            return [UserNote.deserialize_from_db(entry) for entry in cursor]
+
+    def add_user_note(self, title: str, content: str, location: str) -> None:
+        """Add a user_note entry to the DB"""
+        with self.user_write() as write_cursor:
+            write_cursor.execute(
+                'INSERT INTO user_notes(title, content, location, creation_timestamp) VALUES(?, ?, ?, ?)',  # noqa: E501
+                (title, content, location, ts_now()),
+            )
```

Comment:
As I mentioned above this function should gets the cursor's last row id, the identifier, and return it all the way up to the api's response.